docs/administration-guide/gaffer-stores/federated-store.md

```diff
@@ -2,4 +2,406 @@
+instead of the default [`ApplyViewToElementsFunction`](../../reference/operations-guide/federated-operation.md#applyviewtoelementsfunction).
+
+```json title="mergefunctionDefinitions.json"
+{
+  "configuredMergeFunctions": {
+    "uk.gov.gchq.gaffer.operation.impl.get.GetElements": {
+      "class": "uk.gov.gchq.gaffer.federatedstore.util.ConcatenateMergeFunction"
+    },
+    "uk.gov.gchq.gaffer.operation.impl.get.GetAllElements": {
+      "class": "uk.gov.gchq.gaffer.federatedstore.util.ConcatenateMergeFunction"
+    }
+  }
+}
+```
+
+#### Setting default query Graph IDs
+
+By default, when a user submits a query to a Federated Store without using a [Federated Operation](#performing-operations), their query is federated across all graphs they are permitted to read (or write, depending on the query).
+If the Federated Store contains many graphs, this might not be desirable as a default - especially if the graphs do not have compatible schemas.
+It might also be useful for an administrator to configure a defined list of graph IDs to use with these kinds of query, e.g. where a certain primary graph should always be treated as the default for users.
+
+The default graphs to query when a Federated Operation is not used can be set using `storeConfiguredGraphIds`.
+This property takes a path to a JSON file containing a list of Graph IDs to use as the defaults.
+
+The example file given below configures the Federated Store to federate queries across the `GraphA` and `GraphB` graphs by default when a Federated Operation is not used.
+
+```json title="defaultGraphIds.json"
+[
+  "GraphA", "GraphB"
+]
+```
+
+## Adding Graphs
+
+To add a graph to a Federated Store you use the `AddGraph` operation. To load a graph you need to provide:
+
+- Graph ID
+- Graph Schema
+- Graph Properties
+
+!!! note "Duplicate Graph IDs"
+    You can't add a graph with an ID which is already in use, you will need to explicitly [remove the old graph first](#removing-graphs).
+
+A Schema & Properties are not required if `parentSchemaId`/`parentPropertiesId` are instead used to retrieve this information from a Graph Library.
+
+You can limit user access to when adding sub-graphs, see [Limiting Access](#limit-access-with-authentication).
+User specified store properties are allowed by default, but this can be disallowed if desired, see [limit custom properties](#limit-custom-properties).
+
+```json title="AddGraph Operation"
+{
+  "class" : "AddGraph",
+  "graphId" : "AnotherGraph",
+  "schema" : {
+    ... Schema in JSON format
+  },
+  "storeProperties" : {
+    ... Store Properties in JSON format
+  }
+}
+```
+
+??? example "AddGraph with the Java API"
+
+    ```java
+    AddGraph addAnotherGraph = new AddGraph.Builder()
+            .graphId("AnotherGraph")
+            .schema(mySchema)
+            .storeProperties(myStoreProperties)
+            .build();
+    federatedGraph.execute(addAnotherGraph, user);
+    ```
```

Comment:
[This was picked up on already](https://github.com/gchq/gaffer-doc/pull/416#discussion_r1352786961). This will be done when these examples are moved into their own page in the reference guide.